libbeat/publisher/queue/feature.go

```diff
@@ -27,17 +27,12 @@ import (
 // Namespace is the feature namespace for queue definition.
 var Namespace = "libbeat.queue"
 
-// Global queue type registry for configuring and loading a queue instance
-// via common.Config
-var queueReg = map[string]Factory{}
-
 // RegisterType registers a new queue type.
 func RegisterType(name string, fn Factory) {
```

Comment:
Looks like we can remove RegisterType

Reply:
We keep it for backward compatibility. :(